source/api/commands/as.md

```diff
@@ -46,22 +45,44 @@ The name of the alias to be referenced later within a {% url `cy.get()` get %} o
 
 ## DOM Element
 
+Aliasing a DOM element and then {% url getting get %} that aliased element.
+
 ```javascript
-cy.route('PUT', /^\/users\/\d+/, 'fixture:user').as('userPut')
-cy.get('form').submit()
-cy.wait('@userPut')
-  .its('url').should('contain', 'users')
+it('disables on click', function () {
+  cy.get('button[type=submit]').as('submitBtn')
+  cy.get('@submitBtn').click().should('be.disabled')
+})
 ```
 
 ## Route
 
+Aliasing a {% url route %} and then {% url waiting wait %} on that aliased route.
+
 ```javascript
 cy.route('PUT', 'users', 'fx:user').as('userPut')
 cy.get('form').submit()
 cy.wait('@userPut')
   .its('url').should('contain', 'users')
 ```
 
+## Fixture
+
+Aliasing a {% url fixture %} and then accessing it via `this`.
+
+```javascript
+beforeEach(function () {
+  cy.fixture('users-admins.json').as('admins');
+});
+
+it('the users fixture is bound to this.admins', function () {
+  cy.log(`There are ${this.admins.length} administrators.`);
+});
+```
+
+{% note warning %}
+Note the use of the standard function syntax. Using {% url 'arrow functions' https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions %} to access aliases via `this` won't work because of {% url 'the lexical binding' https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_separate_this %} of `this`.
+{% endnote %}
```

Comment:
Here and elsewhere in this PR I've started hyperlinking more frequently. I noticed that you -- Jennifer -- have started doing this in recent PRs so I'm following your lead here. I dunno if hyperlinking to the MDN docs as an authoritative source is cool: steer me if not. I find the MDN docs to be good reference material.

Reply:
Yes! Awesome, we love MDN docs and are not shy about offering general good JS advice when relevant.